frontend/app/player/Player.ts

```diff
@@ -44,6 +44,10 @@ export const INITIAL_STATE = {
   inspectorMode: false,
   live: false,
   livePlay: false,
+  liveTimeTravel: false,
+  lastRecordedMessageTime: 0,
+  timeTravelStart: 0,
+  assistStart: 0,
```

Comment:
please declare it in the lowest component  in the ierarchy where you use it (AssistManager.ts)